logbook-servlet/src/main/java/org/zalando/logbook/servlet/LogbookFilter.java

```diff
@@ -30,7 +30,8 @@
      * Unique per instance so we don't accidentally share stages between filter
      * instances in the same chain.
      */
-    private final String name = ResponseProcessingStage.class.getName() + "-" + UUID.randomUUID();
+    private final String responseProcessingStageName = ResponseProcessingStage.class.getName() + "-" + UUID.randomUUID();
+    private final String responseWritingStageName = ResponseWritingStage.class.getName() + "-" + UUID.randomUUID();
```

Comment:
The name suggests that the attribute value would be of type `ResponseWritingStage`, but it seems to be an `AtomicBoolean`.

Reply:
Thanks, I've renamed it to `ResponseProcessingStage-Synchronization-<UUID>`